packages/base/src/types/CalendarSelection.js

```diff
@@ -0,0 +1,17 @@
+import DataType from "./DataType.js";
+
+const CalendarSelections = {
+	Single: "Single",
+	Multiple: "Multiple",
+	Range: "Range",
+};
+
+class CalendarSelection extends DataType {
+	static isValid(value) {
+		return !!CalendarSelections[value];
+	}
+}
+
+CalendarSelection.generataTypeAcessors(CalendarSelections);
+
```

Comment:
typo: generatA->generatE

Reply:
This method is inherited from the DateType class with the corresponding name.
Not sure I can change that method name.

Reply:
I will fix it everywhere, it is a typo, should be generate

Reply:
https://github.com/SAP/ui5-webcomponents/pull/2500

Reply:
@NHristov-sap  @unazko It is generatE in the master now, thanks for spotting this typo

Reply:
Done